purpose: busybox sed -i expects the argument directly attached, no space is
allowed.
diff --git a/generate-24px-versions.sh b/generate-24px-versions.sh
index 4d73a83..4448776 100644
--- a/generate-24px-versions.sh
+++ b/generate-24px-versions.sh
@@ -15,7 +15,7 @@ case $(sed --help 2>&1) in
     *)
         echo "Using POSIX-style sed arguments"
         sed_i () {
-            sed -i '' "$@"
+            sed -i'' "$@"
         }
         ;;
 esac
